model/utils.go

```diff
@@ -124,7 +124,12 @@ func (m *StringMap) Scan(value interface{}) error {
 
 // Value converts StringMap to database value
 func (m StringMap) Value() (driver.Value, error) {
-	return json.Marshal(m)
+	j, err := json.Marshal(m)
+	if err != nil {
+		return nil, err
+	}
+	// non utf8 characters are not supported
```

Comment:
In the comment I'd add a link to the ticket so people know why we are doing this.